pkgs/by-name/an/angular-language-server/package.nix

```diff
@@ -0,0 +1,105 @@
+{ lib
+, stdenvNoCC
+, fetchYarnDeps
+, fetchFromGitHub
+, nodejs
+, typescript
+, yarnConfigHook
+, yarnBuildHook
+, makeBinaryWrapper
+, runCommand
+, angular-language-server
+, writeShellApplication
+, curl
+, common-updater-scripts
+, jq
+, gnused
+, pcre
+, gawk
+}:
+
+stdenvNoCC.mkDerivation (finalAttrs: {
+  pname = "angular-language-server";
+  version = "18.2.0";
+  src = fetchFromGitHub {
+    owner = "angular";
+    repo = "vscode-ng-language-service";
+    rev = "v${finalAttrs.version}";
+    hash = "sha256-9+WWKvy/Vu4k0BzJwPEme+9+eDPI1QP0+Ds1CbErCN8=";
+  };
+  offlineCache = fetchYarnDeps {
+    yarnLock = "${finalAttrs.src}/yarn.lock";
+    hash = "sha256-N0N0XbNQRN7SHkilzo/xNlmn9U/T/WL5x8ttTqUmXl0=";
+  };
+
+  nativeBuildInputs = [
+    yarnConfigHook
+    yarnBuildHook
+    nodejs
+    makeBinaryWrapper
+  ];
+
+  buildInputs = [
+    typescript
+  ];
+
+  yarnBuildScript = "compile";
+
+  installPhase = ''
+    install -Dm755 bazel-bin/vsix_sandbox/server/bin/ngserver $out/bin/ngserver
+    install -Dm755 bazel-bin/vsix_sandbox/server/index.js $out/index.js
+    cp -r bazel-bin/vsix_sandbox/node_modules $out/node_modules
+  '';
```

Comment:
Missing post/pre install hooks?

Reply:
Jop, I forgot them, thank you for reminding me.